src/graphics/Screen.h

```diff
@@ -322,13 +322,19 @@ class Screen : public concurrency::OSThread
         uint8_t last = LASTCHAR; // get last char
         LASTCHAR = ch;
 
-#if defined(OLED_PL)
-
-        switch (last) { // conversion depending on first UTF8-character
+        switch(last) {
         case 0xC2: {
             SKIPREST = false;
             return (uint8_t)ch;
         }
+        }
+
+        // We want to strip out prefix chars for two-byte char formats
+        if (ch == 0xC2)
+            return (uint8_t)0;
```

Comment:
I think this is causing issues with `OLED_PL`, `OLED_UA` and `OLED_RU`. If using those variants, `switch(last)` is being closed early here, leaving their `case` statements floating loose in the method.

Reply:
You're right, added the switches back where they belong